server/controllers/officeControlers.js

```diff
@@ -106,7 +120,14 @@ class officeManagement {
     }
 
     deleteOffice(req,res) {
-        office.deleteOffice(req.params.id)
+        
+        if(parseInt(req.params.id).toString() == "NaN") {
```

Comment:
Why are you converting that params into string for you to check if it's an integer. there is some other javascript method that can help you do that. you can use typeof or Number.isInterger(). checkout how they are used and choose one you can use.

Reply:
params are by default string thats why am converting them back into string to check if they are valid